app-admin/ruri/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=ruri
+PKGSEC=admin
+PKGDEP="libcap libseccomp"
+PKGDES="A light linux container with zero runtime dependency"
```

Comment:
Please be more descriptive and less commercial-like:

A self-contained Linux container implementation

Reply:
Ok, since this description comes from the github repository description, I will fix it as soon as possible.